README.md

```diff
@@ -517,7 +527,7 @@ Building the program with BLAS support may lead to some performance improvements
     ```bash
     CC=/opt/rocm/llvm/bin/clang CXX=/opt/rocm/llvm/bin/clang++ \
         cmake -H. -Bbuild -DLLAMA_HIPBLAS=ON -DAMDGPU_TARGETS=gfx1030 -DCMAKE_BUILD_TYPE=Release \
-        && cmake --build build -- -j 16
+        && cmake --build build --config Release -- -j 16
```

Comment:
Not directly related to this change, but do you know what the `-H.` is meant to do here? According to the man page, `-H` is "Print usage information and exit".

Reply:
Ah, looks like `cmake -H` will do that but `-H` is ignored when there are other args... and `-H.` is treated as `-H .` == `-H -S .`: I've now removed them all.